File: Data/Counter.cs
namespace Data
{
    public class Counter
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public bool EnableODN { get; set; }

        public int SortOrder { get; set; }

        public int UnitConvertCoefficient { get; set; }


        public int FlatId { get; set; }

        public virtual Flat Flat { get; set; }


        public int CounterTypeId { get; set; }

        public virtual CounterType CounterType { get; set; }


        public int? CounterTarifId { get; set; }

        public virtual CounterTarif CounterTarif { get; set; }
    }
}

File: Comunal/app/models/counter.ts
export class Counter {
    Id: number;
    Selected: boolean;
    TarifCount: number;
    IconPath: string;

    Tarif1: number;
    Limit1: number;
    Tarif2: number;
    Limit2: number;
    Tarif3: number;
    EditTarifOnly: boolean;
    StartReading: number;

    constructor(
        public Name: string = null,
        public FlatId: number = 0,
        public CounterTypeId: number = null,
        public EnableODN: boolean = false,
        public SortOrder: boolean = true,
        public UnitConvertCoefficient: number = 1,
        public CounterTarifId: number | null = null
        ) { }
}